### DISCUSSION TITLE
Testing LruCache

### DISCUSSION THREAD

**Discussion by ezeparziale** [caching-tools/next-shared-cache#223] [Q&A]
I'm using the Redis stack example, as outlined in the following documentation: [https://caching-tools.github.io/next-shared-cache/redis-stack](https://caching-tools.github.io/next-shared-cache/redis-stack)

I'm testing all the features, but when I shut down Redis to test the LRU Cache, my page crashes.

I thought that when Redis is not available, the LRU cache would work, but I can't test this scenario. Any idea?

**Comment by better-salmon** [comment]
Thank you for this information! I will check this scenario as soon as possible.

**Comment by better-salmon** [comment]
I have identified a bug and some oversights in the package. I am working on a fix and tests, which should be ready soon. Please stay tuned!

**Comment by better-salmon** [comment]
Hello @ezeparziale! I apologize for the delay. I have resolved #230 in version `0.6.2`. Moving forward, I will create specialized tests for such scenarios.

**Reply by ezeparziale** [reply]
I upgraded my test repo at https://github.com/ezeparziale/nextjs-k8s to version 0.6.2, and now everything is working correctly.
When Redis is down, the local cache works fine.

Thanks for your prompt reply and fix
